### ISSUE TITLE
Cannot read property 'style' of undefined

### ISSUE THREAD

**Issue by markkesty** [ng2-ui/ng2-auto-complete#71]
On line 152 of ng2-auto-complete.directive.ts, there is a reference to component.inputEl. While component is defined, inputEl is not and so I get the following console error: 

core.umd.js:2844 TypeError: Cannot read property 'style' of undefined
    at Ng2AutoCompleteDirective.styleAutoCompleteDropdown (ng2-auto-complete.directive.ts:152)
    at ZoneDelegate.invokeTask (zone.js:236)
    at Object.onInvokeTask (core.umd.js:5967)
    at ZoneDelegate.invokeTask (zone.js:235)
    at Zone.runTask (zone.js:136)
    at ZoneTask.invoke (zone.js:304)

**Comment by allenhwkim** [comment]
Thanks for reporting this. However this seems not possible for me because Ng2AutoCompleteComponent must have input tag in it and it is assigned with ngOnInit, https://github.com/ng2-ui/ng2-auto-complete/blob/master/src/ng2-auto-complete.component.ts#L152

Without a plunker example, I will consider this as an environmental issue.

**Comment by markkesty** [comment]
It probably is an environmental issue. I created a Plunker and it worked beautifully. The same code failed in my application. **My application is a hybrid Angular 1 and Angular 2 site but I'm using your module in the Angular 2 side.**

Any ideas? I'm close to giving up and trying another component or writing my own.

**Comment by markkesty** [comment]
OK, I was able to successfully make a Plunker of a hybrid Angular 1 and Angular 2 site and I added your component to it. I am using the most simple form of the test page - Array of Strings - since the problem happens there as well and it doesn't involve any http calls or anything fancy like that. Hopefully you'll be able to debug this now and see why it isn't working in a hybrid application.

[https://plnkr.co/edit/dpPaCy?p=preview](url)
